### ISSUE TITLE
chests

### ISSUE THREAD

**Issue by openrsc** [Open-RSC/Game#1725]
Submitted by Discord user: zappa

chests stay open uptill time to loot again, not sure if all chests but the nature chest and the one next to it are the same

**Comment by twothousandandone** [comment]
So it's the nature chest and coin chest next to it that seems to be hanging open too long?

**Comment by kInGkRiStOf** [comment]
Yes, I was there with him during that one, we didn't check any others.
